packages/perseus/src/widgets/__shared__/score-noop.test.ts

```diff
@@ -1,8 +1,8 @@
-import noopValidator from "./noop-validator";
+import scoreNoop from "./score-noop";
 
-describe("noop-validator", () => {
+describe("score-noop", () => {
     it("returns a pointless object", () => {
-        const result = noopValidator();
+        const result = scoreNoop();
```

Comment:
nit: `score[Widget]` makes sense, but I don't think `scoreNoop` really makes sense. I think `noopScorer` is better?

Reply:
I think I was going for consistency with the naming of the scoring related functions, but that seems fine with me. Might be a good idea to keep the file name the way it is to conform to the eslint rules though. What do you think?

Reply:
We talked in messages and decided to leave it as is so it is consistent and conforms to the eslint rules.